dbt/README.md

```diff
@@ -30,49 +30,74 @@ dbt deps
 
 ## Usage
 
-Make sure you have the virtual environment activated:
+To run dbt commands, make sure you have the virtual environment activated:
 
 ```
 source venv/bin/activate
 ```
 
-Authenticate with AWS MFA if you haven't already today:
+You must also authenticate with AWS using MFA if you haven't already today:
 
 ```
 aws-mfa
 ```
 
-Build the models to create views in our Athena warehouse:
+### Build tables and views
+
+Build the models to create tables and views in our Athena warehouse:
 
 ```
 dbt run
 ```
 
 By default, all `dbt` commands will run against the `dev` environment, which
 namespaces the resources it creates by prefixing target database names with
-your Unix `$USER` name (e.g. `jecochr-default` for the `default` database when
-`dbt` is run on Jean's machine). To instead **run commands against prod**,
+your Unix `$USER` name (e.g. `dev_jecochr_default` for the `default` database
+when `dbt` is run on Jean's machine). To instead **run commands against prod**,
 use the `--target` flag:
 
 ```
 dbt run --target prod
 ```
 
-Generate the documentation:
+### Generate documentation
+
+Note that we configure dbt's [`asset-paths`
+attribute](https://docs.getdbt.com/reference/project-configs/asset-paths) in
+order to link to images in our documentation. Some of those images, like the
+Mermaid diagram defined in `assets/dataflow-diagram.md`, are generated
+automatically during the `deploy-dbt-docs` deployment workflow. To generate
+them locally, make sure you have
+[`mermaid-cli`](https://github.com/mermaid-js/mermaid-cli) installed (we
+recommend a [local
+installation](https://github.com/mermaid-js/mermaid-cli#install-locally)) and
+run the following command:
+
+```bash
+for file in assets/*.mmd; do
+  ./node_modules/.bin/mmdc -i "$file" -o "${file/.mmd/.png}"
+done
+```
```

Comment:
Does this feel like too much to ask of users who want to generate docs? I don't love how it duplicates the command in the workflow, but I don't see a better way of documenting it.

Reply:
Nope! I think this is totally fine. Building docs is almost always a pain.